### PR TITLE
web3signer setup with hashicorp vault

### PR DESCRIPTION
**Pull request by sugh01**
Upload keys in batches from Charon folders to Hashicorp Vault.
Create Secret files for Web3Signing key to consume
Example setup with Charon-prater-1

### CODE REVIEW

File: deploy-cluster-perf-node.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+IFS=','
+read -a vcs <<< "$VC_TYPES"
+node_index=0
+for vc in "${vcs[@]}"
+do
+export NODE_NAME="node$node_index"
+export VC_INDEX="vc$node_index"
+if [ $vc -eq 0 ]; then
+eval "cat <<EOF
+$(<./templates/teku-vc.yaml)
```
Comment (aly-obol) [inline deploy-cluster-perf-node.sh:57]: This needs to be templates/teku-vc-web3signer.yaml

File: create-web3signer-secrets.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (aly-obol) [inline create-web3signer-secrets.sh:1]: is this script a good place to execute the go util to decrypt the keys and upload to Vault?

File: deploy-cluster-perf-node.sh (hunk 1 above)
Comment (sugh01) [reply deploy-cluster-perf-node.sh]: Pushed the change

**Comment by sugh01** [comment]
> I am just concerned that any updates for the perf cluster here will be reset by the performance cluster terraform deployments through the infra repo.

@aly-obol , yes, will move this part to terraform for perf, wanted to debug test that everything is okay with the setup

File: create-web3signer-secrets.sh (hunk 2 above)
Comment (sugh01) [reply create-web3signer-secrets.sh:1]: Yes, added this ticket - https://app.zenhub.com/workspaces/obol-workspace-648c1170ee94656741353327/issues/gh/obolnetwork/charon-k8s-distributed-validator-cluster/121

**Pull request closed without merging**